sway/commands/exec_always.c

```diff
@@ -65,8 +65,9 @@ struct cmd_results *cmd_exec_process(int argc, char **argv) {
 		close(fd[0]);
 		if ((child = fork()) == 0) {
 			close(fd[1]);
-			execl("/bin/sh", "/bin/sh", "-c", cmd, (void *)NULL);
-			_exit(0);
+			execlp("sh", "sh", "-c", cmd, NULL);
```

Comment:
While this should work fine on most platforms, C spec allows `NULL` to be an integer, and `man 3 exec` says that `NULL` should be cast to pointer type.

Reply:
Fair enough, reverted this change.

Reply:
Same applies to all the other `execlp`s (although they were missing a cast before too)

Reply:
Right. I don't care enough to make the change in a separate commit, but feel free to open a PR if you want.